pyscript.core/docs/README.md

```diff
@@ -0,0 +1,481 @@
+
+Custom values for the `type` attribute can also be created which alias (and potential build on top of) existing interpreter types. We include `<script type="py">` (and its `<py-script>` custom element counter-part)  which use the Pyodide interpreter while extending its behavior in specific ways familiar to existing PyScript users (*the `<py-config>` tag, `<py-repl>`, etc*).
+
+  </div>
+</details>
+
+<details>
+  <summary><strong>Target</strong></summary>
+  <div>
+
+When it comes to *strings* or *attributes*, we consider the **target** any valid element's *id* on the page or, in most cases, any valid *CSS* selector.
+
+```html
+<!-- ℹ️ - requires py-script custom type -->
+<script type="py">
+    # target here is a string
+    display('Hello PyScript', target='output')
+</script>
+<div id="output">
+    <!-- will show "Hello PyScript" once the script executes -->
+</div>
+```
+
+When it comes to the `property` or `field` attached to a `<script>` element though, that *id* or *selector* would already be resolved, so that such field would always point at the very same related element.
+
+```html
+<script type="micropython" target="output">
+    from js import document
+    document.currentScript.target.textContent = "Hello";
+</script>
+<div id="output">
+    <!-- will show "Hello" once the script executes -->
+</div>
+```
+
+ℹ️ - Please note that if no `target` attribute is specified, the *script* will automatically create a "_companion element_" when the `target` property/field is accessed for the very first time:
+
+```html
+<script type="micropython">
+    from js import document
+
+    # will create a <script-micropython> element appended
+    # right after the currently executing script
+    document.currentScript.target.textContent = "Hello";
+</script>
+<!--
+    created during previous code execution
+
+    <script-micropython>Hello</script-micropython>
+-->
+```
+
+  </div>
+</details>
+
+<details>
+  <summary><strong>Env</strong></summary>
+  <div>
+
+ℹ️ - This is an **advanced feature** that is worth describing but usually it is not needed for most common use cases.
+
+Mostly due its terseness that plays nicely as attribute's suffix, among its commonly understood meaning, we consider an *env* an identifier that guarantee the used *interpreter* would always be the same and no other interpreters, even if they point at very same project, could interfere with globals, behavior, or what's not.
+
+In few words, every single *env* would spawn a new interpreter dedicated to such env, and global variables defined elsewhere will not affect this "_environment_" and vice-versa, an *env* cannot dictate what will happen to other interpreters.
```

Comment:
I find this description a bit confusing... In general, `env` refers to `enviroment`. In this case, it's something like this:

* an environment is an isolated interpreter instance with a specific configuration (packages and files installed into it)
* these environments are identified by "names"
* the `env` attribute on a script allows users to reference a specific environment by name

NOTE:  (Don't want to be picky... just PTSD about communicating the concept of `Enviroments` in the context of Python package managers (`conda`, `pip`, etc..)

Reply:
`env` here defines a unique environment ... every *env* is a different pyodide or a different MicroPython ... I don't personally know any other terminology to define a runtime/interpreter environment, and *env* plays very well with attributes as it's also very short.

The package manager for the *env* import stuff *only* for that *env*, so which part is not clear or you think should be improved as description? It is, in fact, a unique environment for *micropip* as it won't import anything else in any other pyodide out there that is *not the same env* ... am I misunderstanding your comment or did I really mess up docs in here?